consult.el

```diff
@@ -1106,6 +1108,14 @@ MARKER is the cursor position."
   (let ((inhibit-field-text-motion t))
     (consult--region-with-cursor (line-beginning-position) (line-end-position) marker)))
 
+(defun consult--build-arguments (str)
+  "Return arg as flat list of split strings."
```

Comment:
Please expand the docstring here.

Reply:
Do you have package-lint enabled? It should warn you about the undocumented argument.

Reply:
Done, tell me what you think.